react/features/spot/components/SpotControllerView.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+import { SpotView } from 'jitsi-spot-sdk';
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+
+import { ColorSchemeRegistry } from '../../base/color-scheme';
+import { HeaderWithNavigation, Modal } from '../../base/react';
+import { connect } from '../../base/redux';
+
+import { toggleSpotControllerView } from '../actions';
+
+import styles from './styles';
+
+type Props = {
+    _device: Object,
+    _headerStyles: Object,
+    _visible: boolean,
+    dispatch: Function
+};
+
+/**
+ * Implements a view to display the {@code SpotView} component
+ * of the Spot SDK.
+ */
+class SpotControllerView extends PureComponent<Props> {
+    /**
+     * Instantiates a new instance.
+     *
+     * @inheritdoc
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this._onRequestClose = this._onRequestClose.bind(this);
+    }
+
+    /**
+     * Implements {@code PureComponent#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const { _device, _headerStyles, _visible } = this.props;
+
+        return (
+            <Modal
```

Comment:
Why not a dialog?

Reply:
It's the Jitsi SlidingView, not actually a Modal. Check the import

Reply:
Ah, gotcha, disregard then!